app/components/auth-form.tsx

```diff
@@ -26,14 +26,6 @@ export function AuthForm({ mode }: AuthFormProps) {
     },
   });
 
-  const onSubmit = (e: React.FormEvent<HTMLFormElement>) => {
-    e.preventDefault();
-    form.handleSubmit((data) => {
-      // If validation passes, submit the form programmatically
-      const formData = new FormData(e.currentTarget);
-      e.currentTarget.submit();
-    })(e);
-  };
```

Comment:
this was breaking the login form... I think we should probably switch to conform to. I played with it a little bit but havent figured out how to get validation to run with out the error